src/main/java/com/crowdin/cli/properties/CliProperties.java

```diff
@@ -298,17 +305,25 @@ private static void setDefaultValues(PropertiesBean pb) {
                 }
                 if (StringUtils.isEmpty(fileBean.getTranslation())) {
                     errors.add(RESOURCE_BUNDLE.getString("error_empty_translation_section"));
-                } else if (fileBean.getTranslation().contains("**") && fileBean.getSource() != null && !fileBean.getSource().contains("**")) {
-                    errors.add("error: Translation pattern " + fileBean.getTranslation() + " is not valid. The mask `**` can't be used.\n" +
-                            "When using `**` in 'translation' pattern it will always contain sub-path from 'source' for certain file.");
-                }
-                if (!containsLangPlaceholders(fileBean.getTranslation()) && fileBean.getScheme() == null) {
-                    errors.add("`Translation` section doesn't contain language variables");
+                } else {
+                    if (fileBean.getTranslation().contains("**") && fileBean.getSource() != null && !fileBean.getSource().contains("**")) {
+                        errors.add("error: Translation pattern " + fileBean.getTranslation() + " is not valid. The mask `**` can't be used.\n" +
+                                "When using `**` in 'translation' pattern it will always contain sub-path from 'source' for certain file.");
+                    }
+                    if (!containsLangPlaceholders(fileBean.getTranslation()) && fileBean.getScheme() == null) {
+                        errors.add("`Translation` section doesn't contain language variables");
+                    }
                 }
+
                 String updateOption = fileBean.getUpdateOption();
                 if (updateOption != null && !(updateOption.equals("update_as_unapproved") || updateOption.equals("update_without_changes"))) {
                     errors.add("Parameter 'update_option' in configuration file has unacceptable value");
                 }
+                Integer escQuotes = fileBean.getEscapeQuotes();
+                if (escQuotes != null && (escQuotes < 0 || escQuotes > 3)) {
+                    errors.add("Acceptable values for 'escape_quotes are: 0, 1, 2, 3. Default is 3. " +
+                            "https://support.crowdin.com/configuration-file/#escape-quotes-options-for-properties-file-format");
```

Comment:
https://support.crowdin.com/configuration-file-v3/#escape-quotes-options-for-properties-file-format